cfme/tests/intelligence/reports/test_menus.py

```diff
@@ -116,13 +145,93 @@ def test_add_reports_to_available_reports_menu(appliance, request, group,
         initialEstimate: 1/10h
     """
 
-    custom_report = appliance.collections.reports.create(**custom_report_values)
-    request.addfinalizer(custom_report.delete)
     folder = random.choice(report_menus.get_folders(group))
     sub_folder = random.choice(report_menus.get_subfolders(group, folder))
-    with report_menus.manage_subfolder(group, folder, sub_folder) as selected:
-        selected.available_options.fill(custom_report.menu_name)
-        selected.move_into_button.click()
+    report_menus.move_reports(group, folder, sub_folder, get_custom_report.menu_name)
+    report = appliance.collections.reports.instantiate(
+        type=folder, subtype=sub_folder, menu_name=get_custom_report.menu_name
+    )
+    assert report.exists
+
+
+@pytest.mark.tier(1)
+@test_requirements.report
+@pytest.mark.meta(blockers=[BZ(1670293, forced_streams=["5.9", "5.10"])])
+@pytest.mark.parametrize("group", ["EvmGroup-administrator"])
+def test_reports_manage_report_menu_accordion_with_users(
+    appliance, request, group, get_custom_report, report_menus, rbac_user
+):
+    """
```

Comment:
@valaparthvi 
I think ```request``` is not used in this test. So you can remove it.

Reply:
But fixture `rbac_user` needs that, so cannot remove that.